ansible/cluster-infra.yml

```diff
@@ -12,6 +12,9 @@
         cluster_keypair: "{{ cluster_keypair }}"
         cluster_groups: "{{ cluster_groups }}"
         cluster_net: "{{ cluster_net }}"
+#       node_resource: "instance.yaml"
+        node_resource: "Cluster::Instance"
         cluster_config_drive: "{{ cluster_config_drive | default(false) }}"
+#       cluster_az: "ceph"
```

Comment:
?

Reply:
I don't think this should carry over into the generic playbooks, it's RAL-specific.